docs/en/sql-reference/functions/json-functions.md

```diff
@@ -226,14 +226,72 @@ Example:
 SELECT JSONExtractArrayRaw('{"a": "hello", "b": [-100, 200.0, "hello"]}', 'b') = ['-100', '200.0', '"hello"']'
 ```
 
-[Original article](https://clickhouse.tech/docs/en/query_language/functions/json_functions/) <!--hide-->
+## JSONExtractKeysAndValuesRaw {#json-extract-keys-and-values-raw}
 
-## JSONExtractKeysAndValuesRaw(json\[, indices\_or\_keys…\]) {#jsonextractkeysandvaluesrawjson-indices-or-keys}
+Extracts raw data from a JSON object.
 
-Parses key-value pairs from a JSON and returns an array of such pairs, each value represented as unparsed string.
+**Syntax**
 
-Example:
+``` sql
+JSONExtractKeysAndValuesRaw(json[, path])
+```
+
+**Parameters**
+
+- `json` — [String](../data-types/string.md) with valid JSON.
+- `path` — Optional. Comma-separated indices or keys that specify the path to the inner field in a nested JSON object. Each argument can be either a [string](../data-types/string.md) to get the field by the key or an [integer](../data-types/int-uint.md) to get the N-th field (indexed from 1, negative integers count from the end). If not set, the whole JSON is parsed as the top-level object.
```

Comment:
* As far as I remember `path` can be multiple independent arguments, not a single optional one.
* It's not clear what happens if the `json` is not valid or top-level is an array or a scalar instead of an object.
* Is zero allowed as an index?